### ISSUE TITLE
I think the CORAL loss is not correctly implemented.

### ISSUE THREAD

**Issue by yaox12** [jindongwang/transferlearning#27]
The [coral loss](https://github.com/jindongwang/transferlearning/blob/master/code/distance/coral_loss.py) is not correctly implemented.

First, line 15 should be

```python
loss = loss/(4*d*d)
```

instead of 

```python
loss = loss/(4*d*4)
```

Second, according to [Frobenius Norm](http://mathworld.wolfram.com/FrobeniusNorm.html),  after squaring each element and summing them, we should compute the square root of the sum, not the mean of it. 

In line 14

```python
loss = torch.mean(torch.mul((xc - xct), (xc - xct)))
```

should be 

```python
loss = (xc - xct).pow(2).sum().sqrt()
```

**Comment by yaox12** [comment]
Similar discussion in [this issue](https://github.com/SSARCandy/DeepCORAL/issues/17).

**Comment by jindongwang** [comment]
@yaox12 Thank you for your correcting! I somehow forget to update the code from DeepCORAL. Now I modified accordingly.

**Comment by JiaoJinyang** [comment]
I think 'xm = torch.mean(source, 1, keepdim=True) - source' should be 'xm = torch.mean(source, 0, keepdim=True) - source'

**Comment by jindongwang** [comment]
@JiaoJinyang Thanks for your correction! I found that CORAL loss is wrong and I corrected it.
